src/test/kotlin/uk/gov/dluhc/registercheckerapi/rest/UpdatePendingRegisterCheckIntegrationTest.kt

```diff
@@ -935,10 +947,28 @@ internal class UpdatePendingRegisterCheckIntegrationTest : IntegrationTest() {
             .isCreated
 
         // Then
-        assertMessageSubmittedToSqs(
-            queueUrl = localStackContainerSettings.mappedQueueUrlPostalVoteConfirmRegisterCheckResult,
-            expectedMessageContent = expectedMessageContent
-        )
+        when (sourceType) {
+            SourceType.APPLICATIONS_MINUS_API -> assertMessageSubmittedToSqs(
+                queueUrl = localStackContainerSettings.mappedQueueUrlRegisterCheckResultResponse,
```

Comment:
Is it possible to add the queueUrl to the parameters for the test to clean this bit up a little bit?